### PR TITLE
[google_sign_in] Improve example

### PR DESCRIPTION
**Pull request by kevmoo**
Towards https://github.com/flutter/flutter/issues/78073

### CODE REVIEW

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
@ditman would be a better reviewer for the code change than me.

File: packages/google_sign_in/google_sign_in/example/lib/main.dart [hunk 1]
```diff
@@ -5,20 +5,12 @@
 // ignore_for_file: public_member_api_docs
 
 import 'dart:async';
-import 'dart:convert' show json;
 
-import "package:http/http.dart" as http;
+import 'package:collection/collection.dart';
+import 'package:extension_google_sign_in_as_googleapis_auth/extension_google_sign_in_as_googleapis_auth.dart';
```
[the remark this answers is not in the document]
Comment (kevmoo) [reply packages/google_sign_in/google_sign_in/example/lib/main.dart:10]: Or we could drastically simplify this example to just show someones email.

Then put the "real" example in the extension package.

Thoughts @ditman ?

[the remark this answers is not in the document]
Comment (ditman) [reply packages/google_sign_in/google_sign_in/example/lib/main.dart:10]: I think there was value in the old example by doing a raw request to a google service too.

Do you know of any writeup on how to access a raw google API through flutter that is not covered by googleapis? For example: Google Photos?

[the remark this answers is not in the document]
Comment (kevmoo) [reply packages/google_sign_in/google_sign_in/example/lib/main.dart:10]: I don't know of any write-up, although some of the JSON endpoints are documented.

I can revert this and put it in the extension.

**Comment by kevmoo** [comment]
Closing this out. Will do something else here!

**Pull request closed without merging**